environment/src/main/java/com/sequenceiq/environment/environment/validation/EnvironmentValidatorService.java

```diff
@@ -219,4 +221,18 @@ private Boolean singleInstanceIsOnDemandOrSpot(FreeIpaCreationDto freeIpaCreatio
                 .map(spotPercentage -> spotPercentage == ALL_ON_DEMAND_PERCENTAGE || spotPercentage == ALL_SPOT_PERCENTAGE)
                 .orElse(true);
     }
+
+    public ValidationResult validateStorageLocation(String storageLocation) {
+        ValidationResultBuilder resultBuilder = new ValidationResultBuilder();
+        if (storageLocation != null) {
+            Pattern pattern = Pattern.compile(".*\\s.*");
+            Matcher matcher = pattern.matcher(storageLocation.trim());
+            if (matcher.find()) {
+                resultBuilder.error("You have added some whitespace to the storage location: " + storageLocation);
+            }
+        } else {
+            resultBuilder.error("You don't add a storage location, please provide a valid storage location.");
```

Comment:
You did not specified the storage location, please provide a valid storage location.
Would be nice to see which storage location

Reply:
which storage location? the regexp or valid chars?